bin/dsn.cmake

```diff
@@ -231,7 +231,7 @@ endmacro(ms_replace_compiler_flags)
 function(ms_check_cxx11_support)
     if(UNIX)
         include(CheckCXXCompilerFlag)        
-        CHECK_CXX_COMPILER_FLAG("-std=c++11" COMPILER_SUPPORTS_CXX11)
+        CHECK_CXX_COMPILER_FLAG("-std=c++1y" COMPILER_SUPPORTS_CXX11)
```

Comment:
My bad, it is because of move lambda that we abandon VS 2013 and use VS 2015 last time. I thought it was part of the C++11 features when seeing this change as the change should have been done already last time. You have just fixed a bug:)